src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php

```diff
@@ -74,7 +74,7 @@ public static function alterTable(): void
             $wpdb->query('ALTER TABLE '.static::getTableName().' ADD execution_duration TEXT COLLATE utf8mb4_unicode_ci');
         }
 
-        /* @since 7.6.10 */
+        /* @since 8.0.0 */
```

Comment:
?

Reply:
We don't have 7.6.10, I just assumend it before, but we jumped to 8.0.0 after 7.6.9 so I changed this.